### ISSUE TITLE
Implement text wrapping in script text boxes

### ISSUE THREAD

**Issue by sengtsan** [JosefNemec/Playnite#2915]
### Feature description

Currently text does not wrap in the script text boxes. Instead of horizontal scroll bars, Ideally the text would wrap for improved user experience.  

### Screenshots

![Untitled](https://user-images.githubusercontent.com/63714826/166708892-c5523303-ac4d-4728-8672-7989cd91ff6c.jpg)

**Comment by JosefNemec** [comment]
Hmm I'm not sure about this. I always thought wrapping is generally not desired for code.

**Comment by sengtsan** [comment]
Perhaps a toggle then, to accommodate those who would prefer it?

I am not a practicing coder; i use the script section for generally simple
uses such as screen mode changes, turning things like game mode/HDR off and
on and launching programs (e.g. specialK) for game
compatibility/performance reasons. I like to see everything within the text
window without needing to scroll but agree I might be an unrepresentative
user given my simple use cases.

Thanks for taking the time to respond.

**Comment by darklinkpower** [comment]
Just throwing an opinion.

I think adding text wrapping will introduce issues. Powershell execution is line oriented, as new lines are interpreted as different lines to be executed, one at a time. Introducing text wrapping will make it difficult to understand the code in the text box. It may not be an issue when you use a single line like in your screenshot but it will become an issue when you have two lines or more. For those reasons, I think it would be best for it to remain as it is by using an horizontal scroll bar and you can also make the window bigger to see the whole line completely.

**Comment by sengtsan** [comment]
Makes sense. I see your point and agree leaving it would be for the best.
Thanks for listening and sharing your insight.
